src/OpenCensus.Abstractions/Trace/Propagation/IGetter.cs

```diff
@@ -1,23 +0,0 @@
-﻿// <copyright file="IGetter.cs" company="OpenCensus Authors">
-// Copyright 2018, OpenCensus Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of theLicense at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// </copyright>
-
-namespace OpenCensus.Trace.Propagation
-{
-    public interface IGetter<T>
```

Comment:
nit: we could declare Extractor and Injector delegates instead of type/Action. This would enable nice multicast delegate behavior where we can chain multiple propagators together.

Reply:
hm... propagators chaining has nothing to do with delegates. Please elaborate. 

Yes, I think we need chaining. I just didn't implement it yet

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/delegates/how-to-combine-delegates-multicast-delegates

it's just a good idea to declare a common delegate -instead of action - it could be shared between W3C and Zipkin propagators

Reply:
Getter is a way to read value from http request headers. It will be shared between all propagators. No need to multicast. Multicasting needed on propagator level.